azext_iot/operations/events3/_events.py

```diff
@@ -127,13 +129,34 @@ def _output_msg_kpi(msg):
         event_source = {'event': {}}
 
         event_source['event']['origin'] = origin
-        event_source['event']['payload'] = str(next(msg.get_data()), 'utf8')
+
+        payload = ''
+
+        data = msg.get_data()
+        if data:
+            payload = str(next(data), 'utf8')
+
+        system_props = unicode_binary_map(parse_entity(msg.properties, True))
+
+        ct = content_type
```

Comment:
Ensure char casing is uniform, I didn't see content_type.lower() prior in execution